package.json

```diff
@@ -66,12 +68,9 @@
         "axe-core": "4.2.1",
         "apify-shared": ">=0.5.0",
         "axios": ">=0.21.1",
-        "socket.io": ">=2.4.0",
-        "socket.io-parser": ">=3.4.1",
```

Comment:
The updated `yarn.lock` file now has dependencies on older versions of `socket.io-parser` and `engine.io` than the old resolutions allowed. Is that intended? (it looks like #1251 is the PR that originally added both of those, but unfortunately its description doesn't contain links to the specific CVEs it addressed and CG history only appears to go back 3 months)